weblab/templates/entities/entity_list.html

```diff
@@ -1,6 +1,8 @@
 {% extends "base.html" %}
 {% load entities %}
 
+{% block title %}My {{ type }}s - {% endblock title %}
```

Comment:
Title uses "My" but page heading uses "Your". We should pick one :) The interface isn't very consistent on this front anyway (has both my files / your files) - carried over from v1 I think so might need some thought as to which we use.

Reply:
Good point! I think 'your' is probably better. Do @mirams @MichaelClerx have an opinion?

Reply:
I prefer 'your' !
Alternatively: <user name>'s